monix-execution/shared/src/main/scala/monix/execution/schedulers/TestScheduler.scala

```diff
@@ -155,7 +155,7 @@ final class TestScheduler private (
   }
 
   @tailrec
-  protected override def executeAsync(r: Runnable): Unit = {
+  override def executeAsync(r: Runnable): Unit = {
```

Comment:
I'm unsure why Scala 2 admitted access to this method from non-subclasses, but given that this is a `TestScheduler` it seems low-risk to just make this public.